src/pages/settings/about/session-log-warning-modal/session-log-warning-modal.scss

```diff
@@ -0,0 +1,48 @@
+page-session-log-warning-modal {
```

Comment:
It looks like this css was copied and pasted from another file, which is bad practice and makes code very hard to maintain. Is there a way we can reuse the pre-existing css without having to copy and paste it into a new file?